code/modules/media/jukebox.dm

```diff
@@ -405,20 +427,26 @@ var/global/loopModeNames=list(
 		attack_hand(user)
 
 /obj/machinery/media/jukebox/emag(mob/user)
-	current_song = 0
 	if(!emagged)
-		playlist_id = "emagged"
-		last_reload=world.time
-		playlist=null
-		loop_mode = JUKEMODE_SHUFFLE
-		emagged = 1
-		playing = 1
 		user.visible_message("<span class='warning'>[user.name] slides something into the [src.name]'s card-reader.</span>","<span class='warning'>You short out the [src.name].</span>")
-		update_icon()
-		update_music()
-		return 1
+		wires.CutWireIndex(JUKE_CONFIG)
 	return
 
+/obj/machinery/media/jukebox/proc/short()
```

Comment:
I don't really get how it's very different than pulsing a single wire to break into the armory.

Emags are great items - they allow a lot of things. But I feel like MOST of their behaviors should be expeditors (e.g.: instantly open a door instead of having to find that one wire to hack it) instead of walled behavior (e.g.: you simply cannot access the emagged juke without an emag).

If it becomes an issue of contention, I could maybe come up with a state puzzle involving the power wires that would cause a short, but for only someone willing to be persistent and play around with pulses for a while. Don't forget that pulsing has a 1/8 chance of a rad blast, and cutting has a 5/8 chance of shock.